### PR TITLE
Feat: Actor metrics panels addded to Grafana's Zeebe Dashboard

### LINKED ISSUE #12548: Provide Grafana Dashboards for the Actor metrics
**Is your feature request related to a problem? Please describe.**
I understand that [the Actor metrics](https://github.com/camunda/zeebe/blob/a3d2002a0eb74cc701572b4e02abfeb16d6a48a8/scheduler/src/main/java/io/camunda/zeebe/scheduler/ActorMetrics.java) are in the experimental phase, but it would be nice if there exists the Actor dashboard out of the box 🙂

**Describe the solution you'd like**
A separate file, called, for example, actor.json in [the Grafana dashboards](https://github.com/camunda/zeebe/tree/a3d2002a0eb74cc701572b4e02abfeb16d6a48a8/monitor/grafana) folder.

**Describe alternatives you've considered**
Add these metrics to the existing dashboard, for example, [zeebe.json](https://github.com/camunda/zeebe/blob/a3d2002a0eb74cc701572b4e02abfeb16d6a48a8/monitor/grafana/zeebe.json). But I think that it would be more convenient if we have this in a separate dashboard (some users may decide to not enable these metrics thus the dashboard is not needed for them)

### PR DESCRIPTION
**Pull request by rodrigo-lourenco-lopes**
## Description

Actor Metrics added to Zeebe Grafana's Dashboard.

- Actor task execution latency, which measures the time from start to completion (histogram panel and graph p50 & p99 panel).
- Actor job scheduling latency, measuring the time between the scheduling of the job and its execution (histogram panel and graph p50 & p99 panel).
- Actor task count, measuring the number of times an actor task was executed successfully
- Actor task queue length, measuring the length of a job queue for an actor task.

image

These metrics are set as disabled by default behind the ENABLE_ACTOR_METRICS feature flag.

## Related issues

closes #12548

## Definition of Done

_Not all items need to be done depending on the issue and the pull request._

Code changes:
* [ ] The changes are backwards compatibility with previous versions
* [ ] If it fixes a bug then PRs are created to [backport](https://github.com/camunda/zeebe/compare/stable/0.24...main?expand=1&template=backport_template.md&title=[Backport%200.24]) the fix to the last two minor versions. You can trigger a backport by assigning labels (e.g. `backport stable/1.3`) to the PR, in case that fails you need to create backports manually.

Testing:
* [ ] There are unit/integration tests that verify all acceptance criterias of the issue
* [ ] New tests are written to ensure backwards compatibility with further versions
* [x] The behavior is tested manually
* [ ] The change has been verified by a QA run
* [ ] The impact of the changes is verified by a benchmark

Documentation:
* [ ] The documentation is updated (e.g. BPMN reference, configuration, examples, get-started guides, etc.)
* [ ] If the PR changes how BPMN processes are validated (e.g. support new BPMN element) then the Camunda modeling team should be informed to adjust the BPMN linting.

Other teams:
If the change impacts another team an issue has been created for this team, explaining what they need to do to support this change.
- [ ] [Operate](https://github.com/camunda/operate/issues)
- [ ] [Tasklist](https://github.com/camunda/tasklist/issues)
- [ ] [Web Modeler](https://github.com/camunda/web-modeler/issues)
- [ ] [Desktop Modeler](https://github.com/camunda/camunda-modeler/issues)
- [ ] [Optimize](https://github.com/camunda/camunda-optimize/issues)

Please refer to our [review guidelines](https://github.com/camunda/zeebe/wiki/Pull-Requests-and-Code-Reviews#code-review-guidelines).

### CODE REVIEW

File: monitor/grafana/zeebe.json [hunk 1]
```diff
@@ -14759,6 +14680,421 @@
       ],
       "title": "Backups",
       "type": "row"
+    },
+    {
+      "collapsed": false,
```
Comment (deepthidevaki) [inline monitor/grafana/zeebe.json:14685]: :x: Set it as "collapsed": true

**Review by deepthidevaki — CHANGES_REQUESTED** [review]
:rocket:  Thanks @rodrigo-lourenco-lopes . I have some smaller comments.

1. You should also add the histogram view, which you had previously, in addition to the p50 and p99 view. Histogram view is useful to get the overall picture.
2. You can set the unit for that data for latency/duration metrics. Then the visualization automatically displays it correctly
image

File: monitor/grafana/zeebe.json [hunk 2]
```diff
@@ -110,7 +110,7 @@
     {
       "collapsed": true,
       "datasource": {
-        "uid": "${DS_PROMETHEUS}"
+        "uid": "PBFA97CFB590B2093"
```
Comment (deepthidevaki) [inline monitor/grafana/zeebe.json:113]: I believe these changes are fine as it is auto generated when exporting the panel.

**Review by deepthidevaki — APPROVED** [review]
Thanks you.

**Comment by deepthidevaki** [comment]
@rodrigo-lourenco-lopes Please also link the issue in the PR description so that it gets automatically closed.

**Comment by rodrigo-lourenco-lopes** [comment]
bors merge

**Pull request merged**